purekit/src/test/java/com/virgilsecurity/purekit/protocol/PureTestJava.java

```diff
@@ -824,15 +829,15 @@ void delete_key__new_key__should_delete(String pheServerAddress,
                     byte[] plainText = pure.decrypt(authResult1.getGrant(), null, dataId, cipherText);
                 });
 
-                assertEquals(PureException.ErrorCode.CELL_KEY_NOT_FOUND_IN_STORAGE, e.getErrorCode());
+                assertEquals(PureException.ErrorStatus.CELL_KEY_NOT_FOUND_IN_STORAGE, e.getErrorStatus());
             }
         }
         catch (Exception | ProtocolHttpException e) {
             fail(e);
         }
     }
 
-    // TODO: Test hashes encryption
+    // TODO: Test hashes encryption -> ????
```

Comment:
Add test that proves that password hashed are encrypted, stored and can be decrypted